README.md

```diff
@@ -62,28 +63,13 @@ it is expected that the imported class was defined correctly, in another file, a
       .factory('someFactory', someFactory);
   ```
 
-  The same components are now separated into their own files.
+  The same directives are now separated into their own files.
 
-  ```js
```

Comment:
Yes, that is what I am seeing as well.  But this should not be happening?  I will have to review my `.gitattributes` settings.